### DISCUSSION TITLE
On install

### DISCUSSION THREAD

**Discussion by budchirp** [mason-org/mason.nvim#199] [Q&A]
How can I handle if MasonInstall is successful?

**Comment by zhang-stephen** [comment]
I think you can try [this](https://github.com/zhang-stephen/nvim-lsp-loader/blob/9569bb29109e7bb24674b200f1fd7c123db159e6/lua/nvim-lsp-loader/loader.lua#L45).

And the [docs](https://github.com/williamboman/mason.nvim/blob/main/doc/reference.md#package) from mason.nvim, FYI.
